### DISCUSSION TITLE
trust.d landed, will a rules.d follow?

### DISCUSSION THREAD

**Discussion by jw3** [ctc-oss/fapolicy-analyzer#280] [Developers]
The trust.d support was merged into master in fapolicyd recently.  While this has potential for us, it could also be safely ignored.

Trust is simple, we wait for fapolicyd to build the trust db and we consume that, the daemon does the heavy lifting.  

Rules however; we parse ourselves and parsing the .d structure will add some overhead.

Knowing if rules.d will follow would help as we move into the rules based milestones.

**Comment by radosroka** [comment]
Rules.d was intrduced in https://github.com/linux-application-whitelisting/fapolicyd/pull/162.

**Reply by jw3** [reply]
Thanks @radosroka .  These discussions tend to grow stale.  It's good to bring them up to date :+1: 

We integrated Rules.d over a few PRs, the two main ones here (the associated issues are linked in the PRs).

- https://github.com/ctc-oss/fapolicy-analyzer/pull/436
- https://github.com/ctc-oss/fapolicy-analyzer/pull/533

We also added Trust.d support

- https://github.com/ctc-oss/fapolicy-analyzer/pull/672
